projects/Mallard/src/App.tsx

```diff
@@ -22,6 +22,8 @@ import {
 import { nestProviders } from './helpers/provider'
 import { pushNotifcationRegistration } from './helpers/push-notifications'
 
+// This really needs a comment.
```

Comment:
react thinks `useScreens` is a hook bc it begins with `use`, we can change the import name to fix this